src/ncp/ncp_base.cpp

```diff
@@ -3119,6 +3248,25 @@ ThreadError NcpBase::SetPropertyHandler_PHY_CHAN(uint8_t header, spinel_prop_key
     {
         errorCode = otSetChannel(mInstance, static_cast<uint8_t>(i));
 
+#if OPENTHREAD_ENABLE_RAW_LINK_API
+
+         if (errorCode == kThreadError_None)
+         {
+             // Cache the channel. If the raw link layer isn't enabled yet, the otSetChannel call
+             // doesn't call into the radio layer to set the channel. We will have to do it
+             // manually whenever the radios are enabled and/or raw stream is enabled.
+             mCurReceiveChannel = static_cast<uint8_t>(i);
```

Comment:
Using `i` seems consistent with the rest of the NCP code - I wouldn't worry about it in this PR.